docs/user-guide/tasks/grpc.md

```diff
@@ -29,10 +54,11 @@ In addition, the following fields are defined by this task.
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| protoURL  | string (URL) | URL where the [protobuf file](https://developers.google.com/protocol-buffers) that defines the gRPC service is located. |
-| dataURL  | string (URL) | URL where JSON data to be used in call requests is located. |
-| binaryDataURL  | string (URL) | URL where binary data to be used in call requests is located. |
-| metadataURL  | string (URL) | URL where the JSON metadata data to be used in call requests is located. |
+| protoURL | string (URL) | URL where the [protobuf file](https://developers.google.com/protocol-buffers) that defines the gRPC service is located. |
+| dataURL | string (URL) | URL where JSON data to be used in call requests is located. |
+| binaryDataURL | string (URL) | URL where binary data to be used in call requests is located. |
+| metadataURL | string (URL) | URL where the JSON metadata data to be used in call requests is located. |
+| endpoints | map[string]interface | Used to define multiple gRPC methods to test. |
```

Comment:
I'm not sure how to document `endpoints`. Should I just create a new section for it? I'm having trouble explaining that any one of the previously described parameters can be used in `endpoints`.